lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-cloud_manager-provision.rb

```diff
@@ -1,5 +1,5 @@
 module MiqAeMethodService
-  class MiqAeServiceMiqProvisionCloud < MiqAeServiceMiqProvision
+  class MiqAeServiceManageIQ_Providers_CloudManager_Provision < MiqAeServiceMiqProvision
```

Comment:
Should we use a similar namespace hierarchy here instead of collapsing to a snake case class name?